src/main/scala/firrtl/passes/wiring/WiringUtils.scala

```diff
@@ -115,8 +115,9 @@ object WiringUtils {
   def sinksToSources(sinks: Seq[Named],
                      source: String,
                      i: InstanceGraph):
-      Map[Seq[WDefInstance], Seq[WDefInstance]] = {
-    val owners = new mutable.HashMap[Seq[WDefInstance], Vector[Seq[WDefInstance]]]
+      Seq[(Seq[WDefInstance], Seq[WDefInstance])] = {
```

Comment:
Since the return type is changing, does it make sense to deprecate the old `sinksToSources` and add a new method with a different return type?

Reply:
We can. But it means a lot of code duplication (i.e. this PR is going to add > 2x lines).

Reply:
Maybe see if @jackkoenig has an opinion here.

Reply:
It shouldn't require duplication right? Just deprecate `sinkToSources` and have it call the new method that returns a `Seq` then call `.toMap` on it.

Reply:
OK!